src/core/agent/services/connectionService.ts

```diff
@@ -223,6 +219,17 @@ class ConnectionService extends AgentService {
         notes.push(JSON.parse(connection[key]));
       }
     });
+
+    const historyItems: Array<IpexMessageRecord> = [];
```

Comment:
Why does this have the format of `IpexMessageRecord`? That's a class for storing with our DB. We should just have an interface or type